src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -203,6 +207,69 @@ func (r *Cluster) validateKafkaListeners() field.ErrorList {
 	return allErrs
 }
 
+func (r *Cluster) validatePandaproxyListeners() field.ErrorList {
+	var allErrs field.ErrorList
+	var proxyExternal *PandaproxyAPI
+	kafkaExternal := r.ExternalListener()
+	for i, p := range r.Spec.Configuration.PandaproxyAPI {
+		if !p.External.Enabled {
+			continue
+		}
+		if proxyExternal != nil {
+			allErrs = append(allErrs,
+				field.Invalid(field.NewPath("spec").Child("configuration").Child("pandaproxyApi"),
```

Comment:
I think the field should also include index: `.Child("pandaproxyApi").Index(i)`

Reply:
Added